docs/configuration/arguments.md

```diff
@@ -183,6 +183,16 @@ prefix these flags with `distributor.ha-tracker.`
    The timeout for the etcd connection.
 - `etcd.max-retries`
    The maximum number of retries to do for failed ops.
+- `etcd.tls-enabled`
+   Enable TLS.
+- `etcd.tls-cert-file`
+   The TLS certificate file path.
+- `etcd.tls-key-file`
+   The TLS private key file path.
+- `etcd.tls-client-ca-file`
+   The trusted CA file path.
+- `etcd.tls-insecure-skip-verify`
+   Skip validating server certificate.
```

Comment:
Uhm, server or client? Isn't to skip the verification of the provided `-etcd.tls-cert-file`?

Reply:
This flag skips validation of certificate presented by server, so now the description is misleading. Please revert it back to "Skip validating server certificate."